transformer/requirements-tests.txt

```diff
@@ -1,3 +1,4 @@
 -r requirements.txt
+fhirstore @ git+https://github.com/arkhn/pyfhirstore@0.5.0#egg=fhirstore
```

Comment:
wow, we were still depending on the old version in tests ?

Reply:
No, this line doesn't change anything, it's just that I misunderstood something
Deps for tests are in requirements-dev.txt at the root of the project
I'll remove that

Reply:
ok :+1: